### DISCUSSION TITLE
undefined step

### DISCUSSION THREAD

**Discussion by aydinserbest** [cucumber/common#2185] [General]
![Screenshot 2024-07-16 at 08 24 22](https://github.com/user-attachments/assets/714ebbab-c68d-4277-98ee-39e1e4f0563b)
![Screenshot 2024-07-16 at 08 24 39](https://github.com/user-attachments/assets/b007294e-53d3-41c8-b10a-be6cd2617d8d)
![Screenshot 2024-07-16 at 08 24 56](https://github.com/user-attachments/assets/0127bec3-e75f-4470-8376-81de456d22c3)
![Screenshot 2024-07-16 at 08 25 06](https://github.com/user-attachments/assets/7e9c673a-39ee-4777-a4fc-d7603dfe0ff9)

Hello everyone, I tried working with an embedded list in the scenario example instead of a separate table. The test runs and prints the values. My question is: In the feature file, IntelliJ still shows the warning "Undefined step reference: the destinations should be Brussels, London, Amsterdam" for the step, as if it doesn't match the method in the step definition.

**Comment by aydinserbest** [comment]
If I remove the parameter name and use the method name instead, it matches.
![Screenshot 2024-07-16 at 08 31 55](https://github.com/user-attachments/assets/f6aa4649-21f0-443c-aa09-dd5692695a16)
![Screenshot 2024-07-16 at 08 32 14](https://github.com/user-attachments/assets/d785bd14-2ff6-4fcd-8560-135661bbdd71)

**Reply by mpkorstanje** [reply]
This sounds like something you should report to your IDE provider.

**Reply by aydinserbest** [reply]
Thank you @mpkorstanje
